src/Button/IconButton.tsx

```diff
@@ -12,6 +12,10 @@ const IconButton = forwardRef<HTMLButtonElement, IconButtonProps>((props, forwar
     getBaseStyles(theme),
     getSizeStyles(size, variant, true),
     getVariantStyles(variant, theme),
+    // when `size !== 'medium'`, vertical alignment of the icon is thrown off
+    // because changing the font size draws an em-box that does not match the
+    // bounding box of the SVG
+    {fontSize: 1},
```

Comment:
Should we move this to `getSizeStyles` where fontSize is set?

https://github.com/primer/react/blob/4fd9b74a38f313f829e5d201e9b0d782c5cf7b80/src/Button/styles.ts#L213-L237

Reply:
Good call!